pyproject.toml

```
@@ -122,7 +122,7 @@ docs = [
     "ipywidgets",
     "sphinx-issues",
     "sphinx-copybutton>=0.3.2,!=0.5.1",
-    "jupyterlite-sphinx>=0.8.0",
+    "jupyterlite-sphinx>=0.13.0",
```

```suggestion
    "jupyterlite-sphinx>=0.13.1",
```

`jupyterlite-sphinx` `v0.13.0` is bugged, so don't use it. c.f. https://github.com/jupyterlite/jupyterlite-sphinx/pull/152